app/controllers/organisations_controller.rb

```diff
@@ -73,21 +73,30 @@ def details
   end
 
   def new
-    @resource = Organisation.new
+    @organisation = Organisation.new
+    @rent_periods = helpers.rent_periods_with_checked_attr
     render "new", layout: "application"
   end
 
   def create
-    @resource = Organisation.new(org_params)
-    if @resource.save
-      redirect_to organisations_path
+    selected_rent_periods = params.require(:organisation).permit(rent_periods: [])[:rent_periods].compact_blank
```

Comment:
Would it make sense to extract this as a method?

Reply:
happy to.